target/linux/ramips/dts/mt7620n_hootoo_tm05.dts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Device Tree file for the HooToo TM-05
+ *
+ * Copyright (C) 2020 Russell Morris <rmorris@rkmorris.us>
+ *
+ * This file is licensed under the terms of the GNU General Public
+ * License version 2.  This program is licensed "as is" without any
+ * warranty of any kind, whether express or implied.
+ */
```

Comment:
Please replace that by a proper SPDX license identifier (as in other DTS files).

Reply:
Sure, will do! Learning this as I go along ... 😄. I found this, https://spdx.org/licenses/. If you're good with it, I'll update to "GPL-3.0-or-later", and include the "Standard License Header" from (the bottom of) https://spdx.org/licenses/GPL-3.0-or-later.html#licenseText ... OK?

Reply:
Done!

Reply:
@arrmo needs to be redone

Reply:
Indeed, please use proper SPDX identifier.

Reply:
Checked my local file, and the one in the commit - and it seems to contatin the information from https://spdx.org/licenses/GPL-3.0-or-later.html#licenseText ... is that not what is needed? It seems to match what was done earlier, so a bit confused, sorry.

Reply:
@arrmo I understand your confusion now. What we are looking for is not a license, but a license *identifier*

See this and any other DTS as example

https://spdx.github.io/spdx-spec/appendix-V-using-SPDX-short-identifiers-in-source-files/

Reply:
LOL - ya, and the two I used to base off of (similar HW) were both missing it, so I couldn't figure out what you were after. Too funny.

Do I remove the "stuff" I have at the top now, just replace it with the identifier? That's easy, just want to get the right content there.

Reply:
Please just put the matching identifier (a single line) on top of the file, like done in many other DTS files here.

Reply:
Done - thanks!